lighthouse-core/test/fraggle-rock/scenarios/pptr-test-utils.js

```diff
@@ -7,6 +7,7 @@
 
 import {beforeAll, beforeEach, afterAll, afterEach} from '@jest/globals';
 import puppeteer from 'puppeteer-core';
+import {getChromePath} from 'chrome-launcher';
```

Comment:
We can update more tests than this right?

Doing a search for `process.env.CHROME_PATH` should reveal places where `getChromePath` would be useful (e.g. `benchmark-plus-extras.js`)

Reply:
> We can update more tests than this right?
>
> Doing a search for `process.env.CHROME_PATH` should reveal places where `getChromePath` would be useful (e.g. `benchmark-plus-extras.js`)

yeah I was being conservative because I wasn't sure if folks liked having any of those tests _require_ a set `CHROME_PATH`, but happy to update them